client/src/style/app.scss

```diff
@@ -0,0 +1,7 @@
+
+
+.main-app{
+    background:white;
```

Comment:
I see a `$white` color being defined in `./global.scss`. Shouldn't that one be used? :)

Reply:
I was just playing around with styling; I didn't properly start yet.